src/views/Proposal.vue

```diff
@@ -57,6 +57,14 @@
         :votes="votes"
         :strategies="strategies"
       />
+      <PluginSafeSnapCustomBlock
+          v-if="proposal.plugins?.safeSnap?.txs"
+          :proposalConfig="proposal.plugins.safeSnap"
+          :proposalEnd="proposal.end"
+          :porposalId="id"
```

Comment:
There is some typo here look like it was there before, (it's also on the CustomBlock component):
```suggestion
          :proposalId="id"
```